.stylelintrc.json

```diff
@@ -39,6 +41,17 @@
       "ignoreKeywords": ["inherit", "transparent"]
     }],
 
-    "declaration-empty-line-before": null
+    "declaration-empty-line-before": null,
+
+    "scss/at-extend-no-missing-placeholder": null,
```

Comment:
all these rules are being checked by the new scss shared config `stylelint-config-standard-scss`. Shall we turn on some of them?

In the 4th commit you can see that I kept some of the rules:
- space between operator and operands
- no comment empty line
- doublequotes
- etc.

Reply:
@khamui We had a policy to use the same config for all projects, I think we already did the update in a+, so maybe check there and use same config unless theres something you feel we should add or remove in which case it should be done in both projects